web/packages/core/src/plugin-polyfill.ts

```diff
@@ -168,8 +152,7 @@ class RufflePluginArray {
 export const FLASH_PLUGIN = new RufflePlugin(
     "Shockwave Flash",
     "Shockwave Flash 32.0 r0",
-    "ruffle.js",
-    null
```

Comment:
No reason to have the null not be explicit anymore

Reply:
Per discussion on Discord, seems like the `mimeTypes` parameter is unsued since https://github.com/ruffle-rs/ruffle/commit/5ff7168fe9c2dbe8988a1e7b777eeec8d3da04a8#diff-d8d3724115abb346d049c122ef9ff923fb69e5be9e9f7237896d86d955e575fcL135-L156, and now it's not even usable at all.
So I went ahead and removed it completely.